### PR TITLE
remove useSource

### PR DESCRIPTION
**Pull request by BFalquet**
update `script_funs` to enhance reproducibility if the source code of the package is not available.

thank you for the review

### CODE REVIEW

File: tests/testthat/test-chevron_tlg-methods.R [hunk 1]
```diff
@@ -194,25 +194,27 @@ test_that("script_funs works as expected", {
 
 
 test_that("script_funs generates a valid script", {
-  tmp <- tempfile()
-
-  args_list <- list(
-    arm_var = "ARM"
-  )
+  withr::with_tempfile("tmp", fileext = ".R", {
```
Comment (Melkiades) [inline tests/testthat/test-chevron_tlg-methods.R:197]: the rest of the code is identical right?

File: R/chevron_tlg-S4methods.R [hunk 2]
```diff
@@ -365,7 +365,7 @@ setMethod(
     c(
       "# Edit Preprocessing Function.",
       glue::glue("preprocess({name}) <- "),
-      deparse(preprocess(x), control = c("useSource")),
```
Comment (Melkiades) [inline R/chevron_tlg-S4methods.R:368]: Where source was coming from here?

**Review by Melkiades — APPROVED** [review]
Anyway unlocking this. I do not exactly how source was used before for preprocessing but it seems legit to me! Thanks Benoit

File: tests/testthat/test-chevron_tlg-methods.R (hunk 1 above)
Comment (BFalquet) [reply tests/testthat/test-chevron_tlg-methods.R:197]: Yeah, it is just a cleaner way of using a temporary file in the test.

**Pull request merged**